docs/paper/admin/reference/configuration/bukkit-commands-configuration.mdx

```diff
@@ -0,0 +1,65 @@
+---
+slug: /reference/bukkit-commands-configuration
+---
+
+# Bukkit Commands Configuration
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+:::info
+
+The below YAML shows you the structure and default values for the legacy `commands.yml`.
+
+Click on a leaf node to view the description for that setting.
+
+:::
+
+:::warning
+
+The aliases defined in this legacy file do not support modern features
+- Tab completion
+- Permissions
+
+For these, see `config/command-aliases.yml` (Pending PR merge)
```

Comment:
I think just omit this line

Reply:
Sorry, what line specificly? I'm guessing the one that mentions the other pr, but want to check before I get back to my computer

Reply:
Yeah about seeing a file that doesn’t exist, removing line 24 that is. Don’t worry about doing it in a hurry (idk what timezone you are) I won’t likely merge it until some point tomorrow